### ISSUE TITLE
[Bug]

### ISSUE THREAD

**Issue by igor-cali** [apozas/contactdiary#30]
Long tapping a contact and selecting "create a copy today" sets the new contact in the last year (2020).

**To Reproduce**
1. Go to main screen
2. Long tap on contact and select 'create copy today'
4. See error: the new contact is placed at the bottom of the list, with correct day and month but wrong year (e.g. 01/01/2020)

Expected behaviour: new contact placed in current year (e.g. 2021).

Not tested for Events.

**Comment by apozas** [comment]
Hi! Someone else [noticed this bug](https://github.com/apozas/contactdiary/issues/29) (which is also present in events), and has been fixed in https://github.com/apozas/contactdiary/commit/9083927e264d5c2fb877b1e48c8d6bf8c12d2f66. I already made everything necessary for F-Droid to pick up the update, so hopefully sooner than later the F-Drood version will have this fixed.

I'm proceeding to close the issue. Thanks, and Happy New Year! 😄
